Code/Editor/Core/QtEditorApplication.cpp

```diff
@@ -426,8 +429,16 @@ namespace Editor
 
     void EditorQtApplication::InstallEditorTranslators()
     {
-        m_editorTranslator =        CreateAndInitializeTranslator("editor_en-us.qm", ":/Translations");
-        m_assetBrowserTranslator =  CreateAndInitializeTranslator("assetbrowser_en-us.qm", ":/Translations");
+        QTextCodec::setCodecForLocale(QTextCodec::codecForName("utf-8"));
+
+        QStringList translatorNames = {
+            "Editor"
+        };
+
+        for (QString trans : translatorNames)
```

Comment:
`translaterNames` doesn't really have to be a QStringList:

```
    constexpr const char *translatorNames[] = {
        "English"
    };
    for (const char* translator: translatorNames)
    {
        m_translators.append(AzToolsFramework::CreateAndLoadTranslator(QString(translator)));
    }
```
Should also work